Packs/MicrosoftGraphApplications/Integrations/MicrosoftGraphApplications/MicrosoftGraphApplications.py

```diff
@@ -152,6 +229,134 @@ def list_service_principals_command(ms_client: Client, args: dict) -> CommandRes
     )
 
 
+def validate_service_principal_input(args: dict) -> tuple:
+    """
+    Ensure at least one argument is given.
+
+    Args:
+        args: The arguments were passed with the command.
+
+    Returns:
+        If the two arguments are missing, raise an exception, otherwise return one of them.
```

Comment:
which one is returned? why that one? how is this one selected?

Reply:
fixed the docs. In the demo, I'll show and discuss it